db/migrate/20200922170308_link_case_contacts_to_contact_types.rb

```diff
@@ -0,0 +1,36 @@
+class LinkCaseContactsToContactTypes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :case_contact_contact_types do |t|
+      t.references :case_contact, null: false
+      t.references :contact_type, null: false
+
+      t.timestamps
+    end
+
+    code_to_name_mapping = {
+      attorney: ContactType.where(name: "Attorney").first,
```

Comment:
This caused problems - will be fixed in https://github.com/rubyforgood/casa/issues/1056